sql/conseil.sql

```diff
@@ -96,7 +99,26 @@ CREATE TABLE tezos.accounts_checkpoint (
 
 
 --
--- TOC entry 193 (class 1259 OID 99668)
+-- Name: accounts_history; Type: TABLE; Schema: tezos; Owner: -
+--
+
+CREATE TABLE tezos.accounts_history (
```

Comment:
This is a duplicate entry. Please double check diffs in the future as there's no way this ticket could have introduced this table.